config/migrations/2025/20250717144827-filter-form-concept-schemes/20230808112412-add-concept-to-bestuurseenheden.sparql

```diff
@@ -0,0 +1,15 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+
+INSERT {
+    GRAPH <http://mu.semte.ch/graphs/public> {
+        ?s skos:topConceptOf <http://lblod.data.gift/concept-schemes/7e2b965e-c824-474f-b5d5-b1c115740083>.
+        ?s skos:inScheme <http://lblod.data.gift/concept-schemes/7e2b965e-c824-474f-b5d5-b1c115740083>.
+    }
+} WHERE {
+    GRAPH <http://mu.semte.ch/graphs/public> {
```

Comment:
The assumption is wrong, probably because old data from the past.
Check the query:

```
PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>

SELECT DISTINCT ?label ?poef WHERE {
    GRAPH <http://mu.semte.ch/graphs/public> {
        ?s a besluit:Bestuurseenheid ;
          besluit:classificatie ?class.
        ?s skos:inScheme ?inScheme.

       optional {
        ?inScheme skos:prefLabel ?label.
}
        
        ?class skos:prefLabel ?poef
    }
}
```
and best to clean up accordingly. 
(I am not so sure why need that `FILTER NOT EXISTS` anyway?

Reply:
Anway, the result is that Gemeente and provincies are not showing up in the selectbox